.github/CONTRIBUTING.md

```diff
@@ -64,6 +64,7 @@ These are the few directives we have for project maintainers.
     - Try to get secondary maintainer approval before merging if you are able to.
   - PRs with empty commits intended to generate a changelog.
 - Do not merge PRs that contain content from the [banned content list](./CONTRIBUTING.md#banned-content).
+- Do not close PRs purely for breaking a template if the same information is contained without it.
```

Comment:
A: got a lot of balls etc etc 
B: this is a kinda wordy way of getting this across. I also am not a huge fan of using this prescriptively